app/market/market-set/src/main/kotlin/com/hedvig/android/market/set/SetMarketUseCase.kt

```diff
@@ -28,6 +28,7 @@ internal class SetMarketUseCaseImpl(
       languageService.setLanguage(
         when {
           existingLanguage == null -> market.localLanguage()
+          existingLanguage.isSwedishLanguage() -> Language.SV_SE
```

Comment:
Was this case not covered well by the `else` below? Did we ever get wrong results from it perhaps?